### PR TITLE
Update README - adds GoReport badge

### PR DESCRIPTION
**Pull request by pushkyn**
I think it's good idea - add badge for Go Report

### CODE REVIEW

**Comment by cpanato** [comment]
@pushkyn thanks for this PR, let me know when you sign the CLA

**Comment by pushkyn** [comment]
@cpanato - about CLA, signed

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# mmctl ![CircleCI branch](https://img.shields.io/circleci/project/github/mattermost/mmctl/master.svg)
+# mmctl ![CircleCI branch](https://img.shields.io/circleci/project/github/mattermost/mmctl/master.svg)[![Go Report Card](https://goreportcard.com/badge/github.com/mattermost/mmctl)](https://goreportcard.com/report/github.com/mattermost/mmctl)
```
Comment (mgdelacroix) [inline README.md]: very small nit: can you please put a space between badges?

image

Comment (pushkyn) [reply README.md]: @mgdelacroix sure, updated

**Pull request merged**